### PR TITLE
auth

### PR DESCRIPTION
**Pull request by Arjun-sna**

### CODE REVIEW

File: src/svgs/logo.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
```
Comment (Arjun-sna) [inline src/svgs/logo.js]: Create `assets` folder in root of the project and all images there

File: src/components/Auth.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
```
Comment (Arjun-sna) [inline src/components/Auth.js]: Auth.js can be placed inside a separate folder call modules.

Components folder should contain shared reuseable components

File: src/components/Auth.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    this.state = {};
+  }
+  render() {
+    return (
+      <section className="auth">
+        <img src={imgur} alt="imgur" />
+
+        <AuthButton>Connect with Imgur</AuthButton>
+
+        <FloatingButton handleClick={this.props.history.goBack}>
```
Comment (Arjun-sna) [inline src/components/Auth.js]: Is this required?

File: src/components/AuthButton.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const styles = {
```
Comment (Arjun-sna) [inline src/components/AuthButton.js]: why not using scss file.

File: src/components/Home.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React from "react";
```
Comment (Arjun-sna) [inline src/components/Home.js]: Same as Auth.js

File: src/components/Auth.js (hunk 3 above)
Comment (DivyanshBatham) [reply src/components/Auth.js]: `<FloatingButton>` is not required anymore, will remove that. 
`<AuthButton>` is also not required as of now, but we can make a new `<Button>` component for using the same styles OR we can use class to style the buttons.

File: src/components/AuthButton.js (hunk 4 above)
Comment (DivyanshBatham) [reply src/components/AuthButton.js]: I didn't wanted to define a className, and for isolating the styles to only this component.

File: src/components/Auth.js (hunk 2 above)
Comment (DivyanshBatham) [reply src/components/Auth.js]: Okay

File: src/svgs/logo.js (hunk 1 above)
Comment (DivyanshBatham) [reply src/svgs/logo.js]: I once faced an issue, React wasn't allowing me use assets present outside the src folder.

Anyways, we don't need the folder anymore as I took the svg code from the `logo.svg` and added it to `svgs/logo.js` and exported it. Then whenever we want to use the svg we just add it to the JSX.
Something like:
```
import React from "react";
import logo from "../svgs/logo";
import Auth from "../modules/Auth";

const Navbar = () => (
  <nav>
    {logo}
    <Auth/>
  </nav>
);
export default Navbar;
```

File: src/modules/index.js [hunk 6]
```diff
@@ -0,0 +1 @@
+export { default as Auth } from "./Auth";
```
Comment (Arjun-sna) [inline src/modules/index.js]: you don't need to do this.
Just have index fine in each module

**Pull request merged**